src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -1410,6 +1416,84 @@ class DevTask extends AbstractFeatureTask {
         }
     }
 
+    private List<ProjectModule> getProjectModules() {
+        List<ProjectModule> upstreamProjects = new ArrayList<ProjectModule>();
+
+        HashSet<Project> allProjects = new HashSet<>()
```

Comment:
same comment about `Set<Project>` and creating `HashSet` with `<Project>`.

Reply:
Corrected